### PR TITLE
Revert "Non default"

### PR DESCRIPTION
**Pull request by double-beep**
Reverts double-beep/batch#4

### CODE REVIEW

**Comment by double-beep** [comment]
Merged.

**Comment by double-beep** [comment]
Just for completing this pull request.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,2 @@
 Copyright 2018
 Batch experiments
-Assigned to sb.
-No contributors needed.
-Batch experiments starting soon.
```
Comment (helper-beep) [inline README.md]: Not needed

**Comment by helper-beep** [comment]
Ok.

**Pull request merged**